fjord/feedback/api_views.py

```diff
@@ -31,61 +31,66 @@ def get(self, request):
         search = models.ResponseMappingType.search()
         f = F()
 
-        if 'happy' in request.GET:
-            happy = {'0': False, '1': True}.get(request.GET['happy'], None)
-            if happy is not None:
-                f &= F(happy=happy)
-
-        if 'platforms' in request.GET:
-            platforms = request.GET['platforms'].split(',')
-            if platforms:
-                f &= F(platform__in=platforms)
-
-        if 'locales' in request.GET:
-            locales = request.GET['locales'].split(',')
-            if locales:
-                f &= F(locale__in=locales)
-
-        if 'products' in request.GET:
-            products = request.GET['products'].split(',')
-            if products:
-                f &= F(product__in=products)
-
-                if 'versions' in request.GET:
-                    versions = request.GET['versions'].split(',')
-                    if versions:
-                        f &= F(version__in=versions)
-
-        date_start = smart_date(request.GET.get('date_start', None))
-        date_end = smart_date(request.GET.get('date_end', None))
-        delta = smart_timedelta(request.GET.get('date_delta', None))
-
-        if delta is not None:
-            if date_end is not None:
-                date_start = date_end - delta
-            elif date_start is not None:
-                date_end = date_start + delta
-            else:
-                date_end = date.today()
-                date_start = date_end - delta
-
-        # We restrict public API access to the last 6 months.
-        six_months_ago = date.today() - timedelta(days=180)
-        if date_start:
-            date_start = max(six_months_ago, date_start)
-            f &= F(created__gte=date_start)
-        if date_end:
-            date_end = max(six_months_ago, date_end)
-            f &= F(created__lte=date_end)
-
-        search = search.filter(f)
-
-        search_query = request.GET.get('q', None)
-        if search_query is not None:
-            search = search.query(description__sqs=search_query)
-
-        # FIXME: Probably want to make this specifyable
-        search = search.order_by('-created')
+        if 'id' in request.GET:
+            id_list = request.GET['id'].split(',')
+            id_list = (smart_int(id_) for id_ in id_list)
```

Comment:
I think you forgot add the filter by id to the query. Tests pass because you are passing all the IDs that exist. I think?